src/libaktualizr/uptane/fetcher_test.cc

```diff
@@ -11,13 +11,32 @@
 #include "test_utils.h"
 #include "tuf.h"
 
+static const int pause_after = 10;        // percent
+static const int pause_duration = 10;     // seconds
+static const int download_timeout = 200;  // seconds.
```

Comment:
This looks like a major improvement to this test. But do we really have to wait 10 seconds? Maybe we do, but it seems like a lot of time to spend just waiting.

Reply:
True, that's a debug leftover. I'll reduce it to 1 sec.